README.org

```diff
@@ -18,6 +18,9 @@ new syntax.
 
 ** Differences with old animint
 
+
+06-10-2018: =train_layout= [from ggplot2 - R/panel.R] function is now =g_train_layout=. =train_layout= [from animint2- R/z_facet.R] function is the same. Both are =internal functions=.
```

Comment:
hey @vivekktiwari I moved this to NEWS since README should only contain info which is important for users (not info about internal functions)